src/app/modules/main/app_search/view.nim

```diff
@@ -58,3 +62,16 @@ QtObject:
   proc appSearchCompleted(self: View) {.signal.}
   proc emitAppSearchCompletedSignal*(self: View) =
     self.appSearchCompleted()
+
+  proc chatSearchModel*(self: View): chat_search_model.Model =
+    return self.chatSearchModel
+
+  proc buildChatSearchModel*(self: View) {.slot.} =
+    self.delegate.buildChatSearchModel()
+
+  proc chatSearchModelChanged*(self: View) {.signal.}
```

Comment:
Doesn't really need a notifier signal; it's a pointer that never changes, the contents does